### DISCUSSION TITLE
is this a bug ?

### DISCUSSION THREAD

**Discussion by sachinchaudhary1808** [nvim-neorg/neorg#1449] [General]
so i'm using nixvim and i installed nerog using nix overlay for neorg...  this is my config 
```
require("nvim-treesitter.configs").setup({
    highlight = {
        enable = true,
    },
})

require("neorg").setup({
    load = {
        ["core.defaults"] = {},
        ["core.concealer"] = {},
        ["core.dirman"] = {
            config = {
                workspaces = {
                    notes = "~/notes",
                },
                default_workspace = "notes",
            },
        },
    },
})
vim.wo.foldlevel = 99
vim.wo.conceallevel = 2
```
thing is when i open nerog using telescope and press index  file and it tells me that a swap file found and when i press "r" to recover it i got this error
![image](https://github.com/nvim-neorg/neorg/assets/138745629/d61924aa-6406-423b-933b-d887e41941d7)

this does not happen if i open index.neorg file manually without neorg command eg. "Neorg index"

**Comment by vhyrro** [comment]
This looks like Neovim has a problem with the swapfile for the index file. Open up the index file normally, run `:swapname`, then delete that file. See if that fixes it :)

**Reply by sachinchaudhary1808** [reply]
> This looks like Neovim has a problem with the swapfile for the index file. Open up the index file normally, run `:swapname`, then delete that file. See if that fixes it :)

i tried but it happens again as i said it only happens if i open that swap file using neorg index command

**Reply by vhyrro** [reply]
It's entirely likely that another swapfile is still lurking somewhere. Look through the swap directory and remove all references to an index.norg. That's the only possible solution here. In the worst case nuke the whole swap directory if you know there's nothing super important you need swap files of :)

**Reply by sachinchaudhary1808** [reply]
> It's entirely likely that another swapfile is still lurking somewhere. Look through the swap directory and remove all references to an index.norg. That's the only possible solution here. In the worst case nuke the whole swap directory if you know there's nothing super important you need swap files of :)

let me nuke all this shit

**Reply by sachinchaudhary1808** [reply]
> It's entirely likely that another swapfile is still lurking somewhere. Look through the swap directory and remove all references to an index.norg. That's the only possible solution here. In the worst case nuke the whole swap directory if you know there's nothing super important you need swap files of :)

ok i did and then i created other swap and tried to open index.neorg file using neorg index command and same shit
